src/Sylius/Bundle/UserBundle/Form/Type/GenderType.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\UserBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+
+/**
+ * @author Michał Marcinkowski <michal.marcinkowski@lakion.com>
+ */
+class GenderType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function setDefaultOptions(OptionsResolverInterface $resolver)
+    {
+        $resolver->setDefaults(array(
+            'choices' => array(
+                0 => '',
+                1 => 'sylius.gender.male',
+                2 => 'sylius.gender.female',
```

Comment:
With the modern acceptance of a wide variety of "genders" (define that however you wish) I think it would be wise, and without any technical downside, to store either an enumerated column, or somewhat longer string column, perhaps even configurable.

I can imagine for example an adult toy business wanting to use Sylius and finding that flexibility beneficial.